.github/workflows/csslint.yml

```diff
@@ -0,0 +1,13 @@
+name: Stylesheet Lint
+on: [push, pull_request]
+
+jobs:
+    linters:
+        name: stylelint
+        runs-on: ubuntu-latest
+      
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions-hub/stylelint@master
+        env: 
+            PATTERN: "**/*.css" 
```

Comment:
add a line End of file;

Reply:
Ok I added a line at the end of the file.